src/js/angular-datepicker.js

```diff
@@ -238,7 +238,7 @@
             $scope.monthNumber -= 1;
           }
           //set next month
-          $scope.month = $filter('date')(new Date($scope.year + '/' + $scope.monthNumber + '/01'), 'MMMM');
+          $scope.month = $filter('date')(new Date($scope.year, $scope.monthNumber - 1), 'MMMM');
```

Comment:
Sorry i don't understand the -1 can you please just clarify this change so that we can merge